src/helm/benchmark/scenarios/lex_glue_scenario.py

```diff
@@ -0,0 +1,175 @@
+import random
+from pathlib import Path
+from typing import List
+
+import datasets
+from datasets import load_dataset
+
+from .scenario import Scenario, Instance, Reference, CORRECT_TAG, TRAIN_SPLIT, VALID_SPLIT, TEST_SPLIT
+
+# SLTC: Single Class Text Classification
+# MLTC: Multi Class Text Classification
+# QA: Question Answering
+task_code_mapping = {
```

Comment:
nit: `TASK_CODE_MAPPING` as per [PEP 8](https://peps.python.org/pep-0008/#constants). Same for `task_max_train_instances_mapping` and `instructions`

Also same for those variables in lextreme; sorry I missed them before.

Reply:
Thanks, adapted it.